.github/workflows/build.yml

```diff
@@ -19,4 +28,4 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           BRANCH: gh-pages # The branch the action should deploy to.
           FOLDER: dist/acnh-directory # The folder the action should deploy.
-          CLEAN: false
+          CLEAN: true
```

Comment:
I actually need this to be false, as the manually-created `CNAME` and `404.html` files in the `gh-pages` branch need to be retained. This can be set to true if there's a way to ignore those files, or to have them created as part of the build process.